include/mergeSort.h

```diff
@@ -0,0 +1,20 @@
+#ifndef MERGESORT_HPP
+#define MERGESORT_HPP
+#include "arraylist.h"
+using namespace std;
+class mergeSort
+{
+
+public:
+    mergeSort();
+    ~mergeSort();
+
+    arraylist<pair<string, float>> sortArray(arraylist<pair<string, float>> array);
+
+private:
+
+    arraylist<pair<string, float>> merge(arraylist<pair<string, float>>  array1, arraylist<pair<string, float>>  array2);
```

Comment:
Does it not make sense to have this be a template class and have the pair struct override the binary operators or is it not worth the hassle as that's the only type we'll be sorting?